mscore/inspector/inspectorBase.cpp

```diff
@@ -655,6 +671,29 @@ void InspectorBase::resetToStyle()
       score->endCmd();
       }
 
+//---------------------------------------------------------
+//   eventFilter
+///   This blocks scrolling on a scrollable thing when not in focus.
+///   `watched` should be a QComboBox or QAbstractSpinBox.
+///   If this event filter is on any non-QWidget, it will crash.
+//---------------------------------------------------------
+
+bool InspectorScrollPreventer::eventFilter(QObject* watched, QEvent* event)
+      {
+      if (event->type() != QEvent::Wheel) {
+            return QObject::eventFilter(watched, event);
+            }
+
+      if (!qobject_cast<QWidget*>(watched)->hasFocus()) {
+            return true;
+            }
```

Comment:
Hmm, these braces aren't really needed...

Reply:
I know they're not, but there's no harm in having them.

Reply:
except wasting space and violating the MuseScore coding standard ;-)

Reply:
I think we need to have a discussion about this style rule, because I think we'd be better off without it, although I won't run through the reasons for that now. If I get a moment, I'll update it.

Reply:
Wasting space?
The whole MuseScore directory is 4GB... I wonder how much more it would be without this rule... Probably not much.
![image](https://user-images.githubusercontent.com/35939574/73613116-70feb700-45c0-11ea-9e86-ad2c7d258630.png)

Reply:
I'm not talking about disk space, but about screen real estate, you can view less code without scrolling

Reply:
I don't know about you, but I really don't think we should keep unnecessary things. The redundant braces don't make the code more clear, nor more beautiful, isn't that so?

Reply:
That's indeed why the MuseScore code style as well as common sense ;-) recommends against those superfluous braces

Reply:
True, but then again, if you want to add code inside the braces, then the diff only includes your new code... but as I said, probably a discussion for elsewhere. For now, I'll comply with your draconian rules! ;)

Reply:
True the diff would then be one line larger, but a) that's not a big deal and b) a one-time issue only. c) It could get solved by having the opening brack on a separate line, but d) that would waste yet another line permantly and e) violate the coding standard :-)

Reply:
The diff would be two lines larger actually, because you have to add a brace to the if statement line as well. So it effectively overwrites the last commit to touch that line with your one.

Other reasons for always using braces is that it's easier to add code to the block when you're testing things.

And finally, it's always explicit what you want to happen - you don't need to rely on the indentation to make sense of it.

Reply:
Fixed in latest push